src/platform.ts

```diff
@@ -70,16 +70,38 @@ export function getDefaultPowerShellPath(
     // Find the path to powershell.exe based on the current platform
     // and the user's desire to run the x86 version of PowerShell
     if (platformDetails.operatingSystem === OperatingSystem.Windows) {
-        if (use32Bit) {
-            powerShellExePath =
-                platformDetails.isOS64Bit && platformDetails.isProcess64Bit
-                    ? SysWow64PowerShellPath
-                    : System32PowerShellPath;
+        const psCoreInstallPath =
+        (!platformDetails.isProcess64Bit ? process.env.ProgramW6432 : process.env.ProgramFiles) + "\\PowerShell";
```

Comment:
negative ternary ops can be dangerous because the `!` is often overlooked. If you can flip this to be:
```suggestion
        (platformDetails.isProcess64Bit ? process.env.ProgramFiles : process.env.ProgramW6432) + "\\PowerShell";
```
it’s a bit more readable

Reply:
Yes, it took me a good 5 minutes to wrap my head around what this was doing.